### PR TITLE
update reference of spring

### PR DESCRIPTION
**Pull request by yiliuTo**
Update reference of README and javdoc for Spring Boot and Spring Cloud packages.

### CODE REVIEW

File: docs-ref-mapping/reference-unified.yml [hunk 1]
```diff
@@ -842,40 +851,87 @@
...
+      - com.azure.spring.cloud.autoconfigure.cloudfoundry*
+      - com.azure.spring.cloud.autoconfigure.eventhub*
+      - com.azure.spring.cloud.autoconfigure.servicebus*
     - name: Cache Starter
       uid: azure.java.sdk.landingpage.services.springcloud.cache
       href: ~/docs-ref-services/{moniker}/spring-cloud-starter-cache-readme.md
     - name: Spring Cloud Context
       uid: azure.java.sdk.landingpage.services.springcloud.context
       href: ~/docs-ref-services/{moniker}/spring-cloud-context-readme.md
+      childre:
```
Comment (scbedd) [inline docs-ref-mapping/reference-unified.yml:868]:
```suggestion
      children:
```

File: docs-ref-mapping/reference-unified.yml [hunk 2]
```diff
@@ -842,40 +851,87 @@
...
+      - com.azure.spring.cloud.context.core*
+      - com.azure.spring.cloud.autoconfigure.context*
+    - name: Spring Cloud Messaging
+      uid: azure.java.sdk.landingpage.services.springcloud.messaging
+      childre:
```
Comment (scbedd) [inline docs-ref-mapping/reference-unified.yml:873]:
```suggestion
      children:
```

File: docs-ref-mapping/reference-unified.yml [hunk 3]
```diff
@@ -842,40 +851,87 @@
...
+      - com.azure.spring.messaging*
+    - name: Spring Cloud Telemetry
+      uid: azure.java.sdk.landingpage.services.springcloud.telemetry
+      childre:
```
Comment (scbedd) [inline docs-ref-mapping/reference-unified.yml:877]:
```suggestion
      children:
```

**Review by scbedd — COMMENTED** [review]
Leaving final approval to @sima-zhu , but other than the comments looks fine to me!

**Review by sima-zhu — APPROVED** [review]
LGTM. Please address Scott's comments before merge

**Pull request merged**